src/io/flutter/utils/FlutterModuleUtils.java

```diff
@@ -361,11 +361,7 @@ public static void setFlutterModuleAndReload(@NotNull Module module, @NotNull Pr
   }
 
   public static void enableDartSDK(@NotNull Module module) {
```

Comment:
I just want to confirm that we don't try to set a flutter sdk if an sdk is already configured?

Reply:
Good point. I added a test to return immediately if `FlutterSdk.getFlutterSdk(module.getProject())` is not null. Confirmed that new projects do not return.